<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QHFontSetting</class>
 <widget class="QDialog" name="QHFontSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>252</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Tai Le</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Font Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Font On Image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string notr="true">MTF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fiMTFSizespinBox">
          <property name="minimum">
           <number>7</number>
          </property>
          <property name="maximum">
           <number>35</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fiMTFBoldcheckBox">
          <property name="text">
           <string>Bold</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string notr="true">SFR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fiSFRSizespinBox">
          <property name="minimum">
           <number>7</number>
          </property>
          <property name="maximum">
           <number>35</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fiSFRBoldcheckBox">
          <property name="text">
           <string>Bold</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string notr="true">TV Line</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fiTVLineSizespinBox">
          <property name="minimum">
           <number>7</number>
          </property>
          <property name="maximum">
           <number>35</number>
          </property>
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fiTVLineBoldcheckBox">
          <property name="text">
           <string>Bold</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Other</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fiOtherSizespinBox">
          <property name="minimum">
           <number>7</number>
          </property>
          <property name="maximum">
           <number>35</number>
          </property>
          <property name="value">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fiOtherBoldcheckBox">
          <property name="text">
           <string>Bold</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="savepushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
